components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -798,37 +795,17 @@ export class TypeORMWorkspaceDBImpl extends TransactionalDBImpl<WorkspaceDB> imp
         });
     }
 
-    public async countUnabortedPrebuildsSince(cloneURL: string, date: Date): Promise<number> {
+    public async countUnabortedPrebuildsSince(projectId: string, date: Date): Promise<number> {
         const abortedState: PrebuiltWorkspaceState = "aborted";
         const repo = await this.getPrebuiltWorkspaceRepo();
 
         let query = repo.createQueryBuilder("pws");
-        query = query.where("pws.cloneURL = :cloneURL", { cloneURL });
+        query = query.where("pws.projectId != :projectId", { projectId });
```

Comment:
should be a match, correct?

Reply:
Yes 🙈

Reply:
fixing via a8400645e656ab9a59e5f9194a5b9376ba368150 in parallel. Shall we create a separate PR?

Reply:
fine with me to put it into your pr